thirdparty/prod/backports.os.ABOUT

```diff
@@ -15,3 +15,6 @@ copyright: Copyright (c) Piët Delport, and Python Software Foundation
 notes: the invalid_utf8_indexes() helper is copied from Bob Ippolito's 
  pyutf8 package (pyutf8/ref.py) and is MIT-licensed
  https://github.com/etrepum/pyutf8/blob/master/pyutf8/ref.py
+ This verion is an advanced patched version fixing the #688 issue
+ and contain the code in the contributed pull request at 
+ https://github.com/pjdelport/backports.os/pull/7
```

Comment:
Ah, I see. This custom version of `backports.os` still has your #7 merged, whereas we should now use a version that has #8 merged. I guess we can wait until #8 is merged upstream and then use a version straight from upstream.

Reply:
@sschuberth yes, I will update this with the latest https://github.com/pjdelport/backports.os/pull/8 that @pjdelport kindly published to Pypi too.